pkgs/top-level/all-packages.nix

```diff
@@ -33594,7 +33594,9 @@ with pkgs;
 
   pgmanage = callPackage ../applications/misc/pgmanage { };
 
-  pgadmin = callPackage ../applications/misc/pgadmin {
+  pgadmin4 = callPackage ../tools/admin/pgadmin { };
+
+  pgadmin3 = callPackage ../tools/admin/pgadmin/3.nix {
```

Comment:
I know this is already merged, but what were the reasons for keeping `pgadmin3` around?

Reply:
in case someone needs it. I think we should get rid of it after 22.05 gets released.

Reply:
> pgadmin3 was last updated in 2017 and currently doesn't build due to an openssl insecurity.

We are basically accumulating literal trash.

Reply:
> in case someone needs it. I think we should get rid of it after 22.05 gets released tho.

What does pgadmin3 provide that pgadmin4 doesn't? This is NixOS unstable, remove insecure software and add a changelog entry if necessary.

Reply:
I say prune abandonware, and leave a release note.

Reply:
In case someone needs it, they can use nixpkgs 21.11, IMO.

The package will be as broken, depending on an openssl that has been EOL since 2019, on any further release as it was in 21.11.

Reply:
I am for removing it with a release note, too.